src/components/Card.js

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { PALETTE } from "../shared";
+
+import Text from "./Text";
+import SubsectionHeader from "./SubsectionHeader";
+
+const CardEl = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+  border-radius: 4px;
+  border: solid 1px ${PALETTE.white80};
+  box-shadow: 0 8px 16px -8px rgba(19, 12, 51, 0.08);
+
+  ${SubsectionHeader} {
+    margin-bottom: 0.5rem;
+  }
+
+  img {
+    display: block;
+    overflow: hidden;
+    width: 3rem;
+    height: 3rem;
+    border-radius: 50%;
+    margin-bottom: 1.5rem;
+  }
+`;
+
+/**
+ * Note: This exports a React component instead of a styled-component.
+ * [Design Mockup](https://app.zeplin.io/project/5d5aecf2918cf74d46363015/screen/5d813f575d1398783f61df5b)
+ */
+
+const Card = ({ title, icon, children }) => (
+  <CardEl>
+    {icon && <img src={icon} alt={title} />}
+    <SubsectionHeader color={PALETTE.black60}>{title}</SubsectionHeader>
+    <Text>{children}</Text>
+  </CardEl>
+);
+
+Card.propTypes = {
+  /**
+   * The title of the card
```

Comment:
Hm are these used by generated docs or something?

Reply:
Yep to display under Description within Props & Methods! @vcarl